composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "topshelfcraft/supersort",
     "description": "...a super-duper sorting function for your Craft templates.",
     "type": "craft-plugin",
-    "version": "4.0.0",
+    "version": "5.0.0",
```

Comment:
Craft 5 doesn't have any breaking api changes, so we could do a minor version bump instead.
https://craftcms.com/docs/5.x/extend/updating-plugins.html#versioning